### ISSUE TITLE
Not working on mobile browser

### ISSUE THREAD

**Issue by singhpappu** [HichemTab-tech/OTP-designer-jquery#2]
Platform: Android 
Browser: Chrome latest version

Input field: Cursor is not switching after filling in, after filling the value by switching manually, getting incorrect value.
![WhatsApp Image 2023-08-16 at 3 52 35 PM](https://github.com/HichemTab-tech/OTP-designer-jquery/assets/25156691/60543d00-83f2-4a96-bc90-09efe3195fd0)

Chrome Debugger 
![WhatsApp Image 2023-08-16 at 3 54 40 PM](https://github.com/HichemTab-tech/OTP-designer-jquery/assets/25156691/47130736-bc46-4d0e-9153-d87ac08cb113)

**Comment by HichemTab-tech** [comment]
Hi @singhpappu,

Thank you for reporting the issue. I appreciate your effort in bringing this to our attention.

After investigating the problem, I found that the issue was caused by a **Chrome bug** related to the `event.key/keyCode` not working as expected on mobile devices. To address this, I made changes to the library's input event handling, which should now resolve the cursor switching and incorrect value problems you experienced.
